api/src/custom_loader.py

```diff
@@ -30,18 +30,7 @@ def update_collection_summaries(self, collection_id: str) -> None:
                 logger.info(f"Updating extents for collection: {collection_id}.")
                 cur.execute(
                     """
-                    UPDATE collections SET
-                    content = content ||
-                    jsonb_build_object(
-                        'extent', jsonb_build_object(
-                            'spatial', jsonb_build_object(
-                                'bbox', collection_bbox(collections.id)
-                            ),
-                            'temporal', jsonb_build_object(
-                                'interval', collection_temporal_extent(collections.id)
-                            )
-                        )
-                    )
+                    UPDATE collections set content = content || pgstac.collection_extent(collections.id)
```

Comment:
For veda-backend we are overriding the default pgstac collection extent with a new user defined function that captures the max end datetime of items in a collection (when this matters is for annual items that have a nominal datetime of January 1 but you want the collection extent to include the full year so a November search will also intersect that item). https://github.com/NASA-IMPACT/veda-backend/blob/develop/database/runtime/handler.py#L260-L268